kubevirt-conversion/entrypoint

```diff
@@ -36,8 +36,15 @@ if ls /dev/v2v-disk* 2>/dev/null ; then
   done
 fi
 
+if [ -f "/data/input/conversion.json" ] ; then
+  ln -s /data/input/conversion.json /var/lib/uci/input.json
+fi
+
+echo "Lising content of /var/lib/uci"
+ls -lR /var/lib/uci
+
 echo "Listing content of /data"
 ls -lR /data
```

Comment:
This has to be also changed e.g. into `ls -lR /data || true` otherwise the entrypoint will terminate in ManageIQ case.